showcase/src/app/home/home.component.html

```diff
@@ -1,100 +1,114 @@
-		<mat-card class="stark-card">
-			<mat-card-header>
-				<mat-card-title fxLayout fxLayoutAlign="center">
-					<h4 translate>Stark Core</h4>
-				</mat-card-title>
-			</mat-card-header>
-			<mat-card-content>
-				<img src="/assets/images/logo/core.png">
-			</mat-card-content>
-		</mat-card>
-	</a>
-	<a href="https://stark.nbb.be/api-docs/stark-ui/latest">
-		<mat-card class="stark-card">
-			<mat-card-header>
-				<mat-card-title fxLayout fxLayoutAlign="center">
-					<h4 translate>Stark UI - Latest</h4>
-				</mat-card-title>
-			</mat-card-header>
-			<mat-card-content>
-				<img src="/assets/images/logo/logo-blue.svg">
-			</mat-card-content>
-		</mat-card>
-	</a>
-	<a href="https://github.com/NationalBankBelgium/stark/tree/gh-pages/api-docs/stark-ui">
-		<mat-card class="stark-card">
-			<mat-card-header>
-				<mat-card-title fxLayout fxLayoutAlign="center">
-					<h4 translate>Stark UI</h4>
-				</mat-card-title>
-			</mat-card-header>
-			<mat-card-content>
-				<img src="/assets/images/logo/logo-ui.png">
-			</mat-card-content>
-		</mat-card>
-	</a>
+<section class="stark-section bottom-margin">
+	<mat-accordion>
+		<mat-expansion-panel>
+			<mat-expansion-panel-header>
+				<mat-panel-title [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_NEWS_TITLE' | translate">
+				</mat-panel-title>
+			</mat-expansion-panel-header>
+			<h2 [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_NEWS_DESCRIPTION' | translate"></h2>
+			<div class="mat-body-2" [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_NEWS_CONTENT' | translate"></div>
+		</mat-expansion-panel>
+	</mat-accordion>
+</section>
+<section class="stark-section bottom-margin">
+	<mat-accordion>
+		<mat-expansion-panel>
+			<mat-expansion-panel-header>
+				<mat-panel-title [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DOC_MAIN_TITLE' | translate">
+				</mat-panel-title>
+			</mat-expansion-panel-header>
+			<h2 [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DOC_MAIN_DESC' | translate"></h2>
+			<div class="mat-body-2">
+				<h2>Stark Core</h2>
+				<div [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DOC_CORE_DESC' | translate">
+				</div>
+				<br>
+				<h2>Stark UI</h2>
+				<div [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DOC_UI_DESC' | translate">
+				</div>
+				<br>
+				<h2>Showcase</h2>
+				<div [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DOC_SHOWCASE_DESC' | translate">
+				</div>
+			</div>
+		</mat-expansion-panel>
+	</mat-accordion>
+</section>
 
-	<a href="https://github.com/NationalBankBelgium/stark/tree/gh-pages/showcase">
-		<mat-card class="stark-card">
-			<mat-card-header>
-				<mat-card-title fxLayout fxLayoutAlign="center">
-					<h4 translate>Showcase</h4>
-				</mat-card-title>
-			</mat-card-header>
-			<mat-card-content>
-				<img src="/assets/images/logo/showcase.png">
-			</mat-card-content>
-		</mat-card>
-	</a>
-</div>
 
+<!-- FIXME put elsewhere as an example of how to use angular flex-layout-->
+<!--<h1 class="mat-display-2" translate>Stark documentation</h1>-->
```

Comment:
Please remove all this in comments since it is no longer necessary and in any case, if we need it we can check the a previous tag of Stark on Github ;)

Reply:
This code should be kept as an example on how to use flex-layout. I suggest to place it in an example file inside the showcase. We can then integrate it later in a `example-viewer` if needed